pytrip/tripexecuter/execparser.py

```diff
@@ -0,0 +1,244 @@
+                 "debug": ("_na", "s")}
+
+    # field specific arguments. {<trip_parameter> : (<handler_method>, <format_specifier>)}
+    _field_args = {"file": ("_na", "s"),
+                   "import": ("_na", "s"),
+                   "export": ("_na", "s"),
+                   "read": ("_na", "s"),
+                   "write": ("_na", "s"),
+                   "list": ("_na", "s"),
+                   "delete": ("_na", "s"),
+                   "display": ("_na", "s"),
+                   "inspect": ("_na", "s"),
+                   "reset": ("_na", "s"),
+                   "new": ("_na", "s"),
+                   "reverseorder": ("_na", "s"),
+                   "target": ("_na", "s"),
+                   "gantry": ("_na", "s"),
+                   "couch": ("_na", "s"),
+                   "chair": ("_na", "s"),
+                   "stereotacticcoordinates": ("_na", "s"),
+                   "fwhm": ("_na", "s"),
+                   "rastersteps": ("_na", "s"),
+                   "raster": ("_na", "s"),  # abbreviated of the above # TODO: can we implement some pointer/alias?
+                   "zsteps": ("_na", "s"),
+                   "beam": ("_na", "s"),
+                   "weight": ("_na", "s"),
+                   "contourextension": ("_na", "s"),
+                   "contourext": ("_na", "s"),  # abbreviated of the above
+                   "doseextension": ("_na", "s"),
+                   "doseext": ("_na", "s"),  # abbreviated of the above
+                   "projectile": ("_na", "s"),
+                   "proj": ("_na", "s"),  # abbreviated of the above
+                   "bev": ("_na", "s"),
+                   "nolateral": ("_na", "s"),
+                   "raw": ("_na", "s"),
+                   "dosemeanlet": ("_na", "s"),
+                   "algorithm": ("_na", "s"),
+                   "bioalgorithm": ("_na", "s"),
+                   "debug": ("_na", "s")}
+
+    def __init__(self, plan):
+        self.plan = plan
+
+    def _parse_exec(self, path):
+        """ Parse an .exec file and store it into self.plan
+
+        :params str path: path to .exec file including file extension
+        """
+        self.folder = os.path.dirname(path)
+        with open(path, "r") as fp:
+            data = fp.read()
+
+        data = data.split("\n")
+
+        for line in data:
+            _key = line.split(" ")[0]  # get first word of line
+            if _key in self._trip_commands:
+                # lookup and call corresponding method
+                logger.debug("Calling self {:s}".format(self._trip_commands[_key]))
+                getattr(self, self._trip_commands[_key])(line)
+
+    @staticmethod
+    def _unpack_arg(_arg):
+        """ Returns the interior of the paranthesis of an argument.
+        Any quotes will be stripped from the subarg.
+        Any whitespaces will be stripped from the subarg or arg.
+
+        example:
+        _arg = "bolus(2.00)"
+
+        will return
+
+        "bolus", "2.00"
```

Comment:
Use doctest for such explanantion:

```
>>> _unpack_arg("bolus(2.00)"
"bolus", "2.00"
```

such doctests are readable and also can be executed as a form of test